lib/perl/Genome/Sys/Lock/NessyBackend.pm

```diff
@@ -178,6 +181,32 @@ sub min_timeout {
     return 5;
 }
 
+sub _translate_resource {
```

Comment:
If you think it is in fact equivalent then I think `Path::Class` is more terse/clear.  You may want to use `subsumes()` instead of `contains()`; `contains()` actually hits the filesystem.